atlas-4S/tpl-MNIInfant/cohort-1/atlas-4S_description.json

```diff
@@ -0,0 +1,143 @@
+{
+    "Authors": [
+        "M. Cieslak",
+        "T. Salo",
+        "E. Feczko",
+        "T.D. Satterthwaite"
+    ],
+    "Curators": [
+        "M. Cieslak",
+        "T. Salo",
+        "E. Feczko",
+        "T.D. Satterthwaite"
+    ],
+    "Description": "A set of non-cortical atlases and the Schaefer cortical atlas, that have been merged together in template space.",
+    "LevelType": "atlas",
+    "License": "CC BY-ND",
+    "Name": "4S: Schaefer Supplemented with Subcortical Structures",
+    "ReferencesAndLinks": [
+        "https://github.com/PennLINC/AtlasPack"
+    ],
+    "SampleSize": "n/a",
+    "SourceAtlases": {
```

Comment:
This isn't in the BEP or the spec, but it seems like we should have something like it.